subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/accessors/CodeGenerator.kt

```diff
@@ -109,12 +108,16 @@ fun accessibleConventionAccessorFor(targetType: String, name: AccessorNameSpec,
     """
         /**
          * Retrieves the [$original][$type] convention.
+         *
+         * @deprecated Use extensions instead.
```

Comment:
💄 Let the deprecation documentation be more explicit and consistent with other places
```suggestion
         * @deprecated The concept of conventions is deprecated. Use extensions instead.
```
This applies to the other generated convention accessors below.